### ISSUE TITLE
Offsets/Signature are broken for TF2 as of Sept. 7th, 2023.

### ISSUE THREAD

**Issue by LShirohana** [0x00000ED/-CS-Source-FIX-Host_Error-CL_PreserveExistingEntity#1]
I just wanted to let you know the above. I actually did not test your CT file, but someone else did and let me know it was broken/not working.

As a result, I went and found a signature instead, and confirmed it works.

`E8 ? ? ? ? 83 C4 08 5D C3 8B 01 5D` This Signature will take you into `CL_PreserveExistingEntity`, and starts at the CALL instruction for host_Error. So, ? ? ? ? are the 4 bytes (32 bit tf2 game) that lead to the function pointer.

I replaced it with Msg from Tier0.dll, which is an export (making it easy to grab the address for that) and swapped the host_Error ptr with Msg and it worked. 

TL;DR apparently your cheat table is outdated (???), use this signature instead.

**Comment by 0x00000ED** [comment]
unfortunately I don’t have a broken demo, but I checked all the offsets and they are up to date

**Comment by 0x00000ED** [comment]
I reworked the script. It uses aobscanmodule now and works both in tf2 and cs:s.
